app/services/foreman_rh_cloud/cert_auth.rb

```diff
@@ -15,7 +15,10 @@ def execute_cloud_request(params)
       final_params = {
         ssl_client_cert: OpenSSL::X509::Certificate.new(certs[:cert]),
         ssl_client_key: OpenSSL::PKey.read(certs[:key]),
-      }.deep_merge(params)
+        headers: {
+          'X-Org-Id' => ForemanRhCloud.with_iop_smart_proxy? ? organization&.label : nil,
```

Comment:
@ianballou , @jeremylenz  we need to use a different identifier, that is unique per the manifest - something like the account ID, or something similar. What would be the best candidate for it?
We can use the label as a backup option for systems that do not have a valid manifest attached to that org.

Reply:
You could use `Organization.current.owner_details['upstreamConsumer']['uuid']`. This will be unique per manifest.

Reply:
For now the agreement is that we wan a per-org differentiation, so I'll stick with the org label for readability.